index.js

```diff
@@ -25,11 +36,19 @@ module.exports = promisedHandlebars
  *   the promised results. This placeholder may not occur in the template or any partial. Neither
  *   my any helper generate the placeholder in to the result. Errors or wrong replacement will
  *   happen otherwise.
+ * @param {Promise} options.Promise the promise implementation to use. Defaults to global.Promise
  * @returns {Handlebars} a modified Handlebars object
  */
 function promisedHandlebars (Handlebars, options) {
   options = options || {}
   options.placeholder = options.placeholder || '\u0001'
+  if (options.Promise) {
```

Comment:
In version 1.0.x, there should be backwards compatibilty, so if no promise constructor is provided, it should default the the one of `Q`. Q should not be loaded if not needed, so I would replace these lines (45-51) by 

`var Promise = options.Promise || require('q').Promise`

If you want a version without `q` as dependency, we have to create a 2.x-branch, although, this probably means more maintenance-overhead.